db/seeds.rb

```diff
@@ -9,3 +9,11 @@
     end
   end
 end
+
+test_track_app = App.find_or_create_by!(name: 'TestTrack') do |app|
+  app.auth_secret = SecureRandom.urlsafe_base64(32)
+end
+
+IdentifierType.find_or_create_by!(name: 'app_id') do |identifier_type|
+  identifier_type.owner_app = test_track_app
+end
```

Comment:
any reason for this syntax instead of `test_track_app.identifier_types.find_or_create_by!(...)`?

Reply:
in case someone created an `app_id` identifier type already, then we would get a duplicate key error. since we really only care that any identifier type named "app_id" exists (and don't care about the owner app), this felt like a safe approach

Reply:
ah yea. good point. sounds good to me.